apps/site/assets/ts/schedule/components/schedule-finder/daily-schedule/DailySchedule.tsx

```diff
@@ -72,39 +72,41 @@ const SchedulesSelect = ({
 
   return (
     <div className="schedule-finder__service-selector">
-      <label htmlFor="service_selector" className="sr-only">
-        Schedules
+      <label>
```

Comment:
Moving the `<select>` to _inside_ the associated `<label>` seems to work more reliably than using the `htmlFor` attribute, so I decided to try that here.